modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -1904,6 +1906,103 @@ public void testGetObjectEntriesAggregationFacets() throws Exception {
 		Assert.assertEquals(facetValue.getNumberOfOccurrences(), (Integer)2);
 	}
 
+	@Test
+	public void testGetRelatedObjectEntriesAccountEntryRestrictions()
+		throws Exception {
+
+		ObjectDefinition objectDefinition = _createObjectDefinition(
+			Arrays.asList(
+				new TextObjectFieldBuilder(
+				).labelMap(
+					LocalizedMapUtil.getLocalizedMap(
+						RandomTestUtil.randomString())
+				).name(
+					"textObjectFieldName"
+				).build()));
+
+		ObjectRelationship objectRelationship1 =
+			_objectRelationshipLocalService.addObjectRelationship(
+				_adminUser.getUserId(),
+				_objectDefinition3.getObjectDefinitionId(),
+				objectDefinition.getObjectDefinitionId(), 0,
+				ObjectRelationshipConstants.DELETION_TYPE_CASCADE,
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+				"oneToManyRelationshipChild",
```

Comment:
If you are defining `_objectDefinition3` as a parent, I suggest:
```suggestion
				"oneToManyRelationshipParentChild",
```